gitpod-io/gitpod
# [dashboard] Update success criteria dashboard

## Description

Update Workspace Team's Success Criteria Dashboard.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/9026

## How to test

Check the [copy of the dashboard](https://grafana.gitpod.io/d/_eCoPry7k/success-criteria-copy?orgId=1&refresh=1m)
Some values changed based on what I exported from the Settings of the dashboard.

## Release Notes

```release-note
None
```

## Documentation
